apps/client/src/routes/authored/[slug]/edit/+page.server.ts

```diff
@@ -1,40 +1,7 @@
-import { parseFormData, validateFormData } from 'bugs-lib/formData';
-import type { Actions, PageServerLoad } from './$types';
-import { z } from 'zod';
-import { updateCourseChapter } from '$lib/data/courses';
-import { fail } from '@sveltejs/kit';
-
-const chapterSchema = z.object({
-  courseId: z.string(),
-  index: z.number(),
-  title: z.string(),
-  description: z.string(),
-});
+import type { Actions, PageServerLoad } from "./$types";
+import { handleDeleteChapter, handleUpdateChapter } from "../../actions";
 
 export const actions: Actions = {
-  updateChapter: async ({ request }) => {
-    const parsedFormData = parseFormData(await request.formData());
-    const {
-      formData: { courseId, index, title, description },
-    } = validateFormData(parsedFormData, chapterSchema);
-
-    try {
-      const result = await updateCourseChapter(
-        courseId,
-        parseInt(index),
-        title,
-        description
-      );
-
-      return {
-        message: 'Course details updated successfully!',
-      };
-    } catch (e) {
-      console.log(e);
-      return fail(400, { message: 'Chapter details updated failed!' });
-    }
-  },
-  deleteChapter: async ({ request }) => {
-
-  },
+  updateChapter: handleUpdateChapter,
```

Comment:
You could rename this `updateChapterAction` and then use the short-hand property syntax.